### ISSUE TITLE
Error: undefined method `build_from_source=' for #<FormulaInstaller:0x00007fe1e5442c98>

### ISSUE THREAD

**Issue by clarkdunson** [Homebrew/homebrew-core#59437]
Home-brew asked me to report: **Please report this issue:** My system seems fine, and I am not, AFAIK, blocked. I opened this issue on your request, and have saved the entire session in a text file, if you need it. Let me know if you need anything else from me.

THANK YOU!

- [Ran, No] ran `brew update` and can still reproduce the problem?
- [Ran, Warnings] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [Ran, Error No logs] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [Ran, See below] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

## What you were trying to do (and why)

`➜  ~ brew install pdf2svg`

## What happened (include command output)

Lots o' stuff, then:

<details>
  <summary>Command output</summary>
  <pre>

```
==> Pouring epstool-3.09.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/epstool/3.09: 8 files, 271.3KB
Removing: /usr/local/Cellar/epstool/3.08... (8 files, 275.2KB)
==> Checking for dependents of upgraded formulae...
==> Reinstalling 1 broken dependent from source:
openssh
Error: undefined method `build_from_source=' for #<FormulaInstaller:0x00007fe1e5442c98>
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/reinstall.rb:36:in `reinstall_formula'
/usr/local/Homebrew/Library/Homebrew/upgrade.rb:220:in `block in check_installed_dependents'
/usr/local/Homebrew/Library/Homebrew/upgrade.rb:219:in `each'
/usr/local/Homebrew/Library/Homebrew/upgrade.rb:219:in `check_installed_dependents'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:266:in `install'
/usr/local/Homebrew/Library/Homebrew/brew.rb:112:in `<main>'

```
  </pre>
</details>

I have been unable to re-produce the issue, I tried.

## What you expected to happen

No Error. 

## Step-by-step reproduction instructions (by running `brew install` commands)

```
➜  ~ brew config
HOMEBREW_VERSION: 2.4.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: c49ef80dd37bcdbbb2d33b321e58a46fdd102267
Last commit: 10 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: b1ec7839029c60abd11b7566ad7f94b3812bffa4
Core tap last commit: 15 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
CPU: octa-core 64-bit kabylake
Homebrew Ruby: 2.6.3 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
Clang: 11.0 build 1100
Git: 2.21.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.54.0 => /usr/bin/curl
Java: 14
macOS: 10.14.6-x86_64
CLT: 11.3.1.0.1.1576735732
Xcode: 11.3.1
```

```
➜  ~ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libtcl8.6.dylib
  /usr/local/lib/libtk8.6.dylib

Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected header files:
  /usr/local/include/fakemysql.h
  /usr/local/include/fakepq.h
  /usr/local/include/fakesql.h
  /usr/local/include/itcl.h
  /usr/local/include/itcl2TclOO.h
  /usr/local/include/itclDecls.h
  /usr/local/include/itclInt.h
  /usr/local/include/itclIntDecls.h
  /usr/local/include/itclMigrate2TclCore.h
  /usr/local/include/itclTclIntStubsFcn.h
  /usr/local/include/mysqlStubs.h
  /usr/local/include/odbcStubs.h
  /usr/local/include/pqStubs.h
  /usr/local/include/tcl.h
  /usr/local/include/tclDecls.h
  /usr/local/include/tclOO.h
  /usr/local/include/tclOODecls.h
  /usr/local/include/tclPlatDecls.h
  /usr/local/include/tclThread.h
  /usr/local/include/tclTomMath.h
  /usr/local/include/tclTomMathDecls.h
  /usr/local/include/tdbc.h
  /usr/local/include/tdbcDecls.h
  /usr/local/include/tdbcInt.h
  /usr/local/include/tk.h
  /usr/local/include/tkDecls.h
  /usr/local/include/tkPlatDecls.h

Warning: Unbrewed .pc files were found in /usr/local/lib/pkgconfig.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected .pc files:
  /usr/local/lib/pkgconfig/tcl.pc
  /usr/local/lib/pkgconfig/tk.pc

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libtclstub8.6.a
  /usr/local/lib/libtkstub8.6.a

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause brews that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  ghostscript
  python@3.8

Warning: Homebrew's sbin was not found in your PATH but you have installed
formulae that put executables in /usr/local/sbin.
Consider setting the PATH for example like so:
  echo 'export PATH="/usr/local/sbin:$PATH"' >> ~/.zshrc
```

But then I corrected all there issues and still cannot reproduce:

```
➜  ~ brew doctor                                                         
Your system is ready to brew.
```

**Comment by clarkdunson** [comment]
Oops, forgot this:
```
➜  ~ brew gist-logs pdf2svg
Error: No logs.

```

**Comment by Bo98** [comment]
Will be fixed in 2.4.11.

The exact situation this happens isn't too common. It looks like your OpenSSH install is broken, but it couldn't find any update for it so was trying to rebuild it from source (and that part had a bug).

**Comment by MikeMcQuaid** [comment]
(2.4.11 has now been released 🎉 )
